### PR TITLE
Germanic cultures rework part I

### PR DESCRIPTION
**Pull request by IhateTrains**
Part of #601.
- Reworked the mappings to prepare for future additions based on hybridization, divergence and events. Far from done.
- Added Angle, Gepid, Vandilian cultures.
- Added `language_gothic` language for East Germanics.
- Added `heritage_east_germanic` language for East Germanics.

Loc is missing, can add in part II.

### CODE REVIEW

File: ImperatorToCK3/Data_Files/blankMod/output/common/culture/pillars/IR_heritage.txt [hunk 1]
```diff
@@ -30,6 +30,10 @@ heritage_romano_german = {
 	type = heritage
 }
 
+heritage_east_germanic = {
```
Comment (IohannesIohannium) [inline ImperatorToCK3/Data_Files/blankMod/output/common/culture/pillars/IR_heritage.txt:33]: Remember to localize this

File: ImperatorToCK3/Data_Files/blankMod/output/common/culture/pillars/IRToCK3_language.txt [hunk 2]
```diff
@@ -0,0 +1,17 @@
+language_gothic = { # https://en.wikipedia.org/wiki/East_Germanic_languages
```
Comment (IohannesIohannium) [inline ImperatorToCK3/Data_Files/blankMod/output/common/culture/pillars/IRToCK3_language.txt:1]: This should also be localized

File: ImperatorToCK3/Data_Files/blankMod/output/common/culture/cultures/IRToCK3_west_germanic_new.txt [hunk 3]
```diff
@@ -1,4 +1,35 @@
-## CONVERTER
+### CONVERTER ###
+angle = { # https://en.wikipedia.org/wiki/Angles
```
Comment (IohannesIohannium) [inline ImperatorToCK3/Data_Files/blankMod/output/common/culture/cultures/IRToCK3_west_germanic_new.txt:2]: This needs localization

File: ImperatorToCK3/Data_Files/blankMod/output/common/culture/cultures/IRToCK3_east_germanic.txt [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+		35 = caucasian_brown_hair
+		25 = caucasian_dark_hair
+	}
+	coa_gfx = { german_group_coa_gfx western_coa_gfx } 
+	building_gfx = { western_building_gfx } 
+	clothing_gfx = { dde_hre_clothing_gfx western_clothing_gfx } 
+	unit_gfx = { western_unit_gfx } 
+}
+
+vandilian = { # https://en.wikipedia.org/wiki/East_Germanic_peoples
```
Comment (IohannesIohannium) [inline ImperatorToCK3/Data_Files/blankMod/output/common/culture/cultures/IRToCK3_east_germanic.txt:70]: This needs localization

File: ImperatorToCK3/Data_Files/blankMod/output/common/culture/cultures/IRToCK3_east_germanic.txt [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+		25 = caucasian_dark_hair
+	}
+
+	coa_gfx = { german_group_coa_gfx western_coa_gfx } 
+	building_gfx = { western_building_gfx } 
+	clothing_gfx = { dde_hre_clothing_gfx western_clothing_gfx } 
+	unit_gfx = { western_unit_gfx } 
+}
+
+gepid = { # https://en.wikipedia.org/wiki/Gepids
```
Comment (IohannesIohannium) [inline ImperatorToCK3/Data_Files/blankMod/output/common/culture/cultures/IRToCK3_east_germanic.txt:35]: This needs localization

**Review by IohannesIohannium — CHANGES_REQUESTED** [review]

**Review by IohannesIohannium — APPROVED** [review]

**Comment by IhateTrains** [comment]
Thanks for adding locs! Added loc for `language_gothic`.

**Pull request merged**